javascript/javascript1/week3/travelInformation.js

```diff
@@ -0,0 +1,19 @@
+const travelInformation = {
+    speed: 33,
+    destinationDistance: 432,
+};
+
+const travelTime = travelFunction(travelInformation);
+
+function travelFunction(travelInformation) {
```

Comment:
don't use the function before defining it even if it is working. It is better to move line 6 after the function.